nle/env/base.py

```diff
@@ -37,6 +37,99 @@
 
 SKIP_EXCEPTIONS = (b"eat", b"attack", b"direction?", b"pray")
 
+NLE_SPACE_ITEMS = (
```

Comment:
Any reason for not making this a dict already? Immutability?

Reply:
Exactly. Modifying the default object seems dangerous.